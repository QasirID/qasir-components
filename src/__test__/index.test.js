import * as QasirComponents from '..';

describe('Qasir component', () => {
  it('Should Exports all components', () => {
    expect(QasirComponents).toMatchSnapshot();
  });
});
