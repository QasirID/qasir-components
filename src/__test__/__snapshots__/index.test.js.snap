// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Qasir component Should Exports all components 1`] = `
Object {
  "Button": [Function],
}
`;
